functions.php

```diff
@@ -37,8 +37,8 @@ function design_system_include_block_style_variations( $dir_path ) {
 
     // Include specified block style variation files.
     foreach ( glob( $dir_path . '/*.php' ) as $file ) {
-        $baseName = basename( $file, '.php' );
-        if ( in_array( $baseName, $block_style_variation_files, true ) && ! class_exists( $baseName ) ) {
+        $base_name = basename( $file, '.php' );
```

Comment:
i thought we weren't going to do the forloop approach, maybe this is something we should discuss for future.